docs/flux1.md

```diff
@@ -1,44 +1,61 @@
 # Flux1 
 
-This example demonstrates using the Flux1 model to perform tasks such as image generation and mask inpainting based on text prompts. 
+This example demonstrates using the Flux1 model to perform tasks such as image generation and mask inpainting based on text prompts.
+
+### Parameters 
+
+The following parameters can be passed to the model: 
+
+- task: The task to perform using the model - either image generation ("generation") or mask inpainting ("inpainting").
+- prompt: The text prompt describing the desired modifications.
+- height: The height in pixels of the generated image. Defaults to 1024 for best results.
+- width: The width in pixels of the generated image. Defaults to 1024 for best results.
+- num_inference_steps: The number of inference steps to perform. Defaults to 28.
+- guidance_scale: Guidance scale as defined in Classifier-Free Diffusion Guidance. Defaults to 3.5.
+- num_images_per_prompt: The number of images to generate per prompt. Defaults to 1.
+- max_sequence_length: Maximum sequence length to use with the prompt. Defaults to 512.
+- seed: Seed for the random number generator. If not provided, a random seed is used.
+
+## Perform image generation
 
 ```python
 import torch
 from PIL import Image
-from vision_agent_tools.models.flux1 import Flux1, Flux1Config, Flux1Task
+from vision_agent_tools.models.flux1 import Flux1, Flux1Task
 
 # To perform image generation
 
-# Set the prompt for image generation
-prompt = "A purple car in a futuristic cityscape"
-
-# Configure the Flux1 model
-flux_config = Flux1Config(
-    height=512,  # Image dimensions
-    width=512,
-    num_inference_steps=10,  # Number of steps for inference, higher means more detail
-)
-
-# Initialize the Flux1 model
-flux_model = Flux1(model_config=flux_config)
+flux_model = Flux1()
 
-# Generate an image from the prompt
 generated_image = flux_model(
-    task=Flux1Task.IMAGE_GENERATION,  # Image Generation Task
-    prompt=prompt,
+    task=Flux1Task.IMAGE_GENERATION,
+    prompt="A purple car in a futuristic cityscape",
+    height=1024,
+    width=1024,
```

Comment:
it's a common good practice to add height and width together, e.g. numpy shape order is (H, W, C)